cloudinit-updates-jenkins-io.tftpl

```diff
@@ -0,0 +1,24 @@
+#cloud-config
+write_files:
+  #config file for puppet agent on this VM (updates.jenkins.io)
+  #not directly at the destination path as it will fail the packet installation
+  #when writing files, do not use /tmp dir as it races with systemd-tmpfiles-clean LP: #1707222. Use /run/somedir instead. : https://cloudinit.readthedocs.io/en/latest/topics/modules.html#runcmd
+
```

Comment:
```suggestion
  # Configuration file for puppet agent on this VM (${hostname})
  # Not directly at the destination path as it would get the the puppet package installation stuck (asking to override)
  # Also, when writing files, do not use /tmp dir as it races with systemd-tmp. Use /run/<somedir> instead (as per. : https://cloudinit.readthedocs.io/en/latest/topics/modules.html#runcmd)
```